libs/hmpb/layout/src/document_svg.tsx

```diff
@@ -138,15 +137,20 @@ export function SvgPage({
   stroke,
   strokeWidth,
   children,
+  fontFamily,
 }: SvgPageProps): JSX.Element {
   return (
     <svg
+      // Disable common ligatures to avoid `fi` being rendered as a ligature.
+      // Note that this CSS only works on the frontend, so I manually removed
+      // the ligatures from the Roboto-Bold font that we use on the backend.
+      style={{ fontVariantLigatures: 'no-common-ligatures' }}
       x={x}
       y={y}
       width={width}
       height={height}
       viewBox={`0 0 ${width} ${height}`}
-      fontFamily={FONT_FAMILY}
+      fontFamily={fontFamily}
```

Comment:
On the frontend, we want to inherit the `Vx Roboto` font family by leaving this undefined. On the backend, we want to be able to specify the `Roboto` font family. This is all temporary since we'll be moving to solely backend rendering soon.